src/Chat.ts

```diff
@@ -102,15 +104,15 @@ export const createSendMessage = (message: TextMessage) => {
 }
 
 const addMessagesToRoom = (allMessages: Array<ImageMessage | TextMessage>) => {
-    allMessages.forEach(message => {
-        const text = message as TextMessage
-        const image = message as ImageMessage
+    allMessages.forEach((message: ImageMessage | TextMessage )=> {
+     const image = message as ImageMessage
```

Comment:
Once again:
- add `type` key that should be `enum` and distinguish message 
 
eg.:
```
if (message.type === MessageType.Text) {
    const textMessage = message as TextMessage
}
```